diff --git a/packages/db/src/schema.rs b/packages/db/src/schema.rs
index 9dc9e73..2e87a18 100644
--- a/packages/db/src/schema.rs
+++ b/packages/db/src/schema.rs
@@ -21,21 +21,21 @@ diesel::table! {
 
 diesel::table! {
     emulator_profiles (id) {
         id -> Int4,
         emulator_id -> Int4,
         name -> Text,
-        supported_extensions -> Array<Nullable<Text>>,
-        custom_args -> Array<Nullable<Text>>,
+        supported_extensions -> Array<Text>,
+        custom_args -> Array<Text>,
         created_at -> Nullable<Timestamptz>,
         updated_at -> Nullable<Timestamptz>,
     }
 }
 
 diesel::table! {
     emulators (id) {
         id -> Int4,
-        supported_platforms -> Array<Nullable<Int4>>,
+        supported_platforms -> Array<Int4>,
         name -> Text,
         save_strategy -> Int4,
         created_at -> Nullable<Timestamptz>,
         updated_at -> Nullable<Timestamptz>,
@@ -84,16 +84,16 @@ diesel::table! {
         cover_url -> Nullable<Text>,
         background_url -> Nullable<Text>,
         icon_url -> Nullable<Text>,
         igdb_id -> Nullable<Int8>,
         created_at -> Nullable<Timestamptz>,
         updated_at -> Nullable<Timestamptz>,
-        links -> Array<Nullable<Text>>,
-        video_urls -> Array<Nullable<Text>>,
-        screenshot_urls -> Array<Nullable<Text>>,
-        artwork_urls -> Array<Nullable<Text>>,
+        links -> Array<Text>,
+        video_urls -> Array<Text>,
+        screenshot_urls -> Array<Text>,
+        artwork_urls -> Array<Text>,
         release_date -> Nullable<Timestamptz>,
         last_played -> Nullable<Timestamptz>,
         minutes_played -> Nullable<Int4>,
     }
 }
 
